.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/**/*
 .expo/*
 npm-debug.*
+.idea/*
+.history/*
```

Comment:
Your site-specific clutter should not be added to the repo-wide `.gitignore`.  Use your local git config, or the private `.git/info/exclude`.